themes/basic_eww/README.md

```diff
@@ -1,4 +1,14 @@
-Important note:
+This is a very basic README merely containing a bunch of heads-up notes for using `eww` with `leftwm`
```

Comment:
I think this file, in particular, is very helpful to have! Might be helpful to maybe add a similar one to all the basic themes? Thanks for doing this!

Reply:
Great idea!
Would we wan't to wait with this PR until I could include them, or merge this right away and add them to the others later?

Reply:
no, lets make that a separate PR.